cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/service/http/HttpHandlerGenerator.java

```diff
@@ -515,10 +515,11 @@ public void visitClassType(String name) {
      * <pre>{@code
      *   public void|BodyConsumer handle(HttpRequest request, HttpResponder responder, ...) {
      *     T handler = getHandler();
-     *     TransactionContext txContext = getTransactionContext();
-     *     DelayedHttpServiceResponder wrappedResponder = wrapResponder(responder, txContext);
+     *     TransactionContext txContext = null;
      *     HttpContentConsumer contentConsumer = null;
      *     try {
+     *       txContext = getTransactionContext();
+     *       DelayedHttpServiceResponder wrappedResponder = wrapResponder(responder, txContext);
```

Comment:
Seems like the comment is not align with the actual code being generated. With the change, the code generated looks like this:

```java
TransactionContext txContext = null;
HttpContentConsumer contentConsumer = null;
DelayedHttpServiceResponder wrappedResponder = null;
try {
  txContext = getTransactionContext();
  wrappedResponder = wrapResponder(responder, txContext);
  ....
}